subgraphs/uniswap-v2/src/common/Price.ts

```diff
@@ -0,0 +1,183 @@
+/* eslint-disable prefer-const */
+import { BigDecimal, Address, BigInt } from '@graphprotocol/graph-ts/index'
+import { factoryContract, UNTRACKED_PAIRS } from './helpers'
+import { LiquidityPool, Token, Bundle } from '../../generated/schema'
+import { ZERO_ADDRESS, BIGDECIMAL_ZERO, BIGDECIMAL_ONE } from '../common/constants'
+
+const WETH_ADDRESS = '0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2'
+const USDC_WETH_PAIR = '0xb4e16d0168e52d35cacd2c6185b44281ec28c9dc' // created 10008355
+const DAI_WETH_PAIR = '0xa478c2975ab1ea89e8196811f51a7b7ade33eb11' // created block 10042267
+const USDT_WETH_PAIR = '0x0d4a11d5eeaac28ec3f61d100daf4d40471f1852' // created block 10093341
```

Comment:
These should go into the `constants` file

Reply:
Moved to constants